src/components/overview/OverviewMain.jsx

```diff
@@ -279,58 +400,69 @@ class overviewMain extends React.Component {
   }
 
   render() {
-    //conditional rendering to make sure everything is loaded
-    var zoomStyling;
     if (this.props.storeState.reviewsMetadata.product_id) {
```

Comment:
I have similar conditional statements in my code. Does this product_id property ever not exist when you are async loading data? I have had bugs from things like that. Something to be aware of.